frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -11,6 +11,13 @@ export interface PayloadAction<Payload> {
   payload: Payload;
 }
 
+const EMPTY_FILTERS: State["selectedFilters"] = {
```

Comment:
Does it make sense to reuse the existing `EMPTY_FILTERS` object [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/2756/files#diff-cf34339b9c061a8efa652b2ec5e97f93ea9d16bb21cc3a69af0578bd0175b910R539)? You wouldn't then have to remember to keep both in sync. (This isn't an option of course if the existing `EMPTY_FILTERS` is mutated somewhere in the code.)